docs/src/main/sphinx/release/release-366.md

```diff
@@ -0,0 +1,43 @@
+# Release 366 (XX Dec 2021)
+
+## General
+         
+* Add support for query retries. This feature can be turned on by setting the `retry-policy` 
```

Comment:
should we mention that the retries are "automatic"? i.e. without any client side intervention?